packages/swift/src/testing.xcodeproj/xcuserdata/stephensewell.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -28,12 +28,12 @@
 		<key>testing (iOS).xcscheme_^#shared#^_</key>
 		<dict>
 			<key>orderHint</key>
-			<integer>36</integer>
+			<integer>34</integer>
```

Comment:
Why the changes in this file? It looks like maybe a local file that should not be committed?

Reply:
no clue, I think these should probably be gitignored, will look into